src/test/java/com/example/networkautomation/NetworkAutomationApplicationTests.java

```diff
@@ -2,12 +2,29 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import static org.hamcrest.Matchers.equalTo;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 
 @SpringBootTest
-class NetworkAutomationApplicationTests {
+@AutoConfigureMockMvc
+public class NetworkAutomationApplicationTests {
+
+    @Autowired
+    private MockMvc mvc;
 
     @Test
-    void contextLoads() {
-    }
+    public void getHello() throws Exception{
+        mvc.perform(MockMvcRequestBuilders.get("/")
+                .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(content().string(equalTo("Welcome to Network Automation")));
 
+    }
```

Comment:
Why did you added the same test in two different file? Aren't you trying to test the same thing in both the file? if yes then you should leave these file as is and do all the testing by opening a new module nammed controller and you have to put all the controller related tests there.